dynamic-mapping-ui/src/configuration/broker-configuration.component.ts

```diff
@@ -95,6 +95,9 @@ export class BrokerConfigurationComponent implements OnInit, OnDestroy {
       outboundMappingEnabled: new FormControl('')
     });
     this.feature = await this.sharedService.getFeatures();
+	if (!this.feature.userHasMappingAdminRole) {
+		this.alert.warning('The configuration on this tab is not editable, as you don\'t have Mapping ADMIN permissions. Please assign Mapping ADMIN permissions to your user.');
```

Comment:
To make it more readable I would change "Mapping ADMIN" to "MAPPING ADMIN" (so all role related info as capital letters)

Reply:
This is fine. Thanks.